src/MongoDB/Monitoring/CommandFailedEvent.c

```diff
@@ -96,7 +96,7 @@ PHP_METHOD(CommandFailedEvent, getOperationId)
 		return;
 	}
 
-	sprintf(int_as_string, "%" PHONGO_LONG_FORMAT, intern->operation_id);
+	sprintf(int_as_string, "%" PRId64, intern->operation_id);
```

Comment:
IIRC, `operation_id` and `request_id` are unsigned 64-bit integers. These should use `PRIu64` instead of `PRId64`.